SPEC.md

```diff
@@ -174,7 +174,7 @@ It also increments the message tree index by the number of commands whose proces
 
 This function should, however, only do so if the processed message counter indicates that all previous messages have already been processed.
 
-Only the coordinator may invoke this function.
+Only the coordinator may invoke this function?
```

Comment:
Looks like only the coordinator can successfully invoke this function because it requires a proof. But we don't have an explicitly `isCoordinator` modifier in the `MACI.sol` contract though.

Reply:
I'll update the spec to address this point!